java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -1098,6 +1119,10 @@ private native void addArmNN(long apiHandle, long nativeHandle, int useArena)
 
     private native void addCoreML(long apiHandle, long nativeHandle, int coreMLFlags)
         throws OrtException;
+
+    private native void sessionOptionsAppendExecutionProvider(
```

Comment:
maybe a name like `addExecutionProvider` would be more consistent with the other method names?
nit: camel case for parameter names po_key, po_val

Reply:
`appendExecutionProvider` is fine with me, it should drop the `sessionOptions` though as it's an instance method on `SessionOptions`.

Reply:
Let's take `addExecutionProvider` to keep addEP similiar things but a more general method